src/test/java/com/checkout/payments/PaymentsClientImplTest.java

```diff
@@ -79,6 +84,41 @@ void shouldRequestPayment() throws ExecutionException, InterruptedException {
 
     }
 
+    @Test
+    void shouldRequestPayment_customerSource() throws ExecutionException, InterruptedException {
+
+        CustomerSource customerSource = new CustomerSource(10, Currency.USD);
+
+        final PaymentRequest request = mock(PaymentRequest.class);
+        request.setSource(customerSource);
```

Comment:
Please replace this mock with a real object - something like `PaymentRequest.builder().source(customSource).build()`. I think it makes more sense in this context.